stable/aws-for-fluent-bit/README.md

```diff
@@ -91,6 +91,15 @@ helm delete aws-for-fluent-bit --namespace kube-system
 | `firehose.timeKey` | Add the timestamp to the record under this key. By default the timestamp from Fluent Bit will not be added to records sent to Kinesis. | |
 | `firehose.timeKeyFormat` | strftime compliant format string for the timestamp; for example, `%Y-%m-%dT%H:%M:%S%z`. This option is used with `time_key`. | |
 | `firehose.extraOutputs` | Append extra outputs with value | `""` |
+| `kinesis_streams.enabled` | Whether this plugin should be enabled or not, [details](https://github.com/aws/amazon-kinesis-streams-for-fluent-bit) | `false` | ✔
+| `kinesis_streams.region` | The AWS region. | ✔
+| `kinesis_streams.stream` | The name of the Kinesis Streams Delivery Stream that you want log records send to. | ✔
+| `kinesis_streams.sts_endpoint` | Custom endpoint for the STS API. `kinesis_streams.sts_endpoint`. | |
```

Comment:
This seems to be missing a few options:
- endpoint
- role_arn

Reply:
Thnak you for review!
I updated readme your comment.